{{cookiecutter.repostory_name}}/dc-prod.yml

```diff
@@ -68,6 +68,8 @@ services:
     command: celery flower --app={{cookiecutter.django_project_name}} --broker="${CELERY_BROKER_URL}" --basic_auth="${CELERY_FLOWER_USER}:${CELERY_FLOWER_PASSWORD}"
     depends_on:
       - celery-worker
+    ports:
+      - 5555:5555
```

Comment:
whoa, so we have an admin/admin available at a public port? that's not a secure default. Generate a password like for postgres, or even better, make sure it only gets hosted on 127.0.0.1 so that we have to tunnel in via ssh to access it

Reply:
Stupid docker.

Ok, make a random password then and lets merge it.